tsconfig.json

```diff
@@ -14,6 +14,7 @@
     "experimentalDecorators": true,
     "emitDecoratorMetadata": true,
     "forceConsistentCasingInFileNames": true,
+    "skipLibCheck": true,
```

Comment:
i know you've talked to me about this change, but can we please document it somewhere?

Reply:
`skipLibCheck` basically it's a recommended rule for typescript since you don't want to traspile and make typing checks for files inside the node_modules or any other lib since you don't have control over that code. You have more info here in the official documentation https://www.typescriptlang.org/tsconfig#skipLibCheck

Reply:
excellent. thank you chonita